_tour/basics.md

```diff
@@ -119,7 +116,7 @@ println(addOne(1)) // 2
 ```
 {% endscalafiddle %}
 
-Functions may take multiple parameters.
+A function can take multiple arguments:
```

Comment:
I am not convinced that _parameters_ should be replaced by _arguments_ given that the nomenclature universally uses the former (e.g. they are "parameter blocks" not "argument blocks").

Reply:
I can change this to "have" multiple parameters.